packages/web-config-server/src/apiV1/index.js

```diff
@@ -56,14 +55,13 @@ export const getRoutesForApiV1 = () => {
     handleWith(OrganisationUnitHandler),
   );
   api.get('/organisationUnitSearch', handleWith(OrgUnitSearchHandler));
-  api.get('/dashboard', handleWith(DashBoardHandler)); // Legacy style dashboards
-  api.get('/view', handleWith(ViewHandler));
+  api.get('/dashboard', handleWith(DashboardHandler)); // Legacy style dashboards
```

Comment:
I will remove this endpoint and also drop `dashboardReport` + `dashboardGroup` in the next PR (and also add Admin Panel tabs for the new tables)